pkg/cluster/cache/factory.go

```diff
@@ -0,0 +1,12 @@
+package cache
+
+import (
+	"github.com/goharbor/harbor-operator/pkg/lcm"
+)
+
+// NewCacheController returns a cache controller.
+// TODO add options
+func NewCacheController() lcm.Controller {
```

Comment:
this func need to define which args need to input.

Reply:
Yup, this need input k8s.GetOptions, which defined in another pr, after the common k8s.GetOptions merged, i will add the option here.